File: Kotlin/3.kt
fun countEven(num: Long): Int {
    var counter = 0
    var tempNum = num
    while (tempNum != 0L) {
        var n = tempNum % 10
        if (n % 2 == 0L && n != 0L) {
            counter++
        }
        tempNum /= 10
    }
    return counter
}

fun main() {
    val totalNums = readLine()!!.toLong()
    for (i in 1..totalNums) {
        val number = readLine()!!.toLong()
        println(countEven(number))
    }
}

File: PHP/3.php
<?
function countEven($num) {
    $counter = 0;
    while ($num != 0) {
        $n = (int)($num % 10);
        if ($n % 2 == 0) {
            $counter += 1;
        }
        $num = (int)($num /= 10);
    }
    return $counter;
}

$n = readline();
for ($i = 0; $i < $n; $i++) {
    $num = readline();
    echo countEven($num) . "\n";
}
?>